src/routes/api/post/comments/[commentId]/replies/+server.ts

```diff
@@ -0,0 +1,15 @@
+import { getDb } from '$lib/server';
+import { loadMoreRepliesToComment } from '$lib/server/services/content';
+import { json, type RequestHandler } from '@sveltejs/kit';
+
+export const GET: RequestHandler = async ({ params: { commentId }, url }) => {
+    const db = await getDb();
+
+    const page = Number(url.searchParams.get('offset'));
```

Comment:
check for `isNaN(page)` and throw a 400 error if it is.

Reply:
done